### ISSUE TITLE
"exclude" parameter not working

### ISSUE THREAD

**Issue by diegocanal** [billerickson/display-posts-shortcode#168]
I am using the following shortcode:
`[display-posts taxonomy="category" tax_term="monografias" taxonomy_2="category" tax_2_term="laboralistas" tax_relation="AND" exclude="3538" image_size="dcr_sidebar_featured_image_thumbnail_2" wrapper="div" wrapper_class="recent-post-widget" include_excerpt="false" include_date="false"]`

I thoroughly tested it (I made sure the post ID is correct using the `id` parameter in another shortcode)

**Comment by billerickson** [comment]
What version of the plugin are you using? The exclude parameter is a new feature only found in the development version (2.8.1) here on GitHub. If you downloaded the plugin from WordPress.org it won't have it (2.8). 

Once all outstanding issues have been taken care of, I'll push the new version to WordPress.org.

**Comment by diegocanal** [comment]
I am using the development version (2.8.1)

**Comment by diegocanal** [comment]
I have found out the reason why it's not working. I long time ago, when the `exclude_current` parameter was still not available I put in my functions.php file the following code:
```
function be_exclude_current_post( $args ) {
	if( is_singular() && !isset( $args['post__in'] ) )
		$args['post__not_in'] = array( get_the_ID() );
	return $args;
}
add_filter( 'display_posts_shortcode_args', 'be_exclude_current_post' );
```
I've removed it and now the `exclude` parameter is working as expected.

Sorry, I should have checked everything in depth before posting.

@billerickson thank you very much for your help and for building and maintaining this fabulous plugin.
